src/components/executive-education-2u/components/CourseSummaryCard.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card, Image, Row, Col,
+} from '@edx/paragon';
+import { numberWithPrecision } from '../../course/data/utils';
+
+const CourseSummaryCard = ({ courseMetadata, enrollmentCompleted }) => (
+  <Card
+    className="mb-4 course-summary"
+    orientation="horizontal"
+  >
+    <Card.Body>
+      <Card.Header
+        title={(
+          <Image
+            src={courseMetadata.organizationImage}
+          />
+        )}
+      />
+      <Card.Section>
+        <Row>
+          <Col xs={12} lg={{ span: 8, offset: 0 }} className="vertical">
+            <Row>
+              <Col xs={12} lg={{ span: 8, offset: 0 }}>
+                <p className="small font-weight-light text-gray-500">{ courseMetadata.organizationName }</p>
+                <p>{ courseMetadata.title }</p>
+              </Col>
+            </Row>
+          </Col>
+          <Col xs={12} lg={{ span: 4, offset: 0 }}>
+            <div className="course-details">
+              <Row>
+                <Col className="small font-weight-light text-gray-500 justify-content-start">{enrollmentCompleted ? 'Start date:' : 'Available start date:'}</Col>
+                <Col className="justify-content-end"><Row className="justify-content-end mr-2.5">{courseMetadata.startDate}</Row></Col>
+              </Row>
+              <Row>
```

Comment:
It looks like the alignment of the values is off from their labels:

<img width="1181" alt="image" src="https://user-images.githubusercontent.com/2828721/231525732-aa6bde5e-d11b-4d38-bcf6-ebc0b5434d7f.png">

From Figma mocks:

<img width="627" alt="image" src="https://user-images.githubusercontent.com/2828721/231525792-d3c32374-1b13-45e5-be6e-cf760081e213.png">

Reply:
Strange, Alignment is looking good on my end, I tested on Chrome, Firefox and Safari.
<img width="1213" alt="Screen Shot 2023-04-13 at 11 41 10 AM" src="https://user-images.githubusercontent.com/7114956/231675544-deaec526-8e71-46bf-a54c-5bde50c272c8.png">

Reply:
I can not find any issue in the code as well as to why it would happen on some machines. Can you try again? or maybe suggest a solution if you can fix it on your end?

Reply:
Interesting. Just pulled down the branch again to try again. I still see the alignment issue, at least when using the `@edx/brand-edx.org` theme, in Chrome. That said, adding an `align-items-center` class name on these `Row` components seems to fix it.

```jsx
<Row className="align-items-center">
```

<img width="1183" alt="image" src="https://user-images.githubusercontent.com/2828721/232473865-7a257e3d-6417-4fe7-9bf3-821540fe8e90.png">